website/content/docs/lambda/registration/manual.mdx

```diff
@@ -0,0 +1,62 @@
+---
+layout: docs
+page_title: Manually Configuring AWS Lambdas
+description: >-
+  Manually configure AWS Lambda services in Consul Service Mesh.
+---
+
+# Manual Configuration
+
+## Register the Service
+
+Store the following in `lambda.json`:
+```json
+{
+  "Node": "lambdas",
+  "SkipNodeUpdate": true,
+  "NodeMeta": {
+    "external-node": "true",
+    "external-probe": "true"
+  },
+  "Service": {
+    "ID": "lambda",
+    "Service": "lambda"
+  }
+}
```

Comment:
A little bit of explanation would clarify here. Are `Service.ID` and `Service.Service` always set to exactly "lambda", or should I change them in some cases?

Even adding "this configuration can be used exactly as shown" would help (if that's the case).